src/modules/70-pipeline/components/PipelineStudio/TemplateBar/__tests__/TemplateBar.test.tsx

```diff
@@ -222,4 +259,17 @@ describe('<TemplateBar /> tests', () => {
     )
     expect(container).toMatchSnapshot()
   })
+
+  test('should render repo and branch name if template linked is remote', async () => {
+    useGetTemplate.mockImplementation(
+      () => ({ data: stageTemplate, refetch: jest.fn(), error: null, loading: false } as any)
+    )
+    const { getByText } = render(
```

Comment:
Better to alias locally for a test, rather than complete file

```suggestion
  const { getByText: getByTxt } = render(
```